Dockerfile

```diff
@@ -14,9 +14,11 @@ COPY entrypoint.sh /
 ENV PATH="/usr/local/bin:${PATH}"
 
 RUN chmod +x /entrypoint.sh
-RUN apk update \
```

Comment:
The salesforcedx official Docker image uses a base ubuntu image so apk needed to be replaced with apt-get